src/interface/swagger.json

```diff
@@ -2114,52 +2250,60 @@
         },
         "QuotesIDPutResponse": {
             "title": "QuotesIDPutResponse",
-            "description": "PUT /quotes/{ID} object",
+            "type": "object",
+            "description": "The object sent in the PUT /quotes/{ID} callback.",
             "required": [
                 "transferAmount",
                 "expiration",
                 "ilpPacket",
                 "condition"
             ],
-            "type": "object",
             "properties": {
                 "transferAmount": {
-                    "$ref": "#/definitions/Money",
-                    "description": "The amount of money that the Payee FSP should receive."
+                    "description": "The amount of money that the Payee FSP should receive.",
+                    "type": "object",
```

Comment:
i dont think type is required as the "Money" definition include type

Reply:
yeah, it isn't required, fixed